pcsx2/gui/FrameForGS.cpp

```diff
@@ -438,7 +438,7 @@ GSFrame::~GSFrame() throw()
 void GSFrame::OnCloseWindow(wxCloseEvent& evt)
 {
 	sApp.OnGsFrameClosed( GetId() );
-	Hide();		// and don't close it.
+	CoreThread.Reset(); // reset thread to avoid suspend status on close event.
```

Comment:
> Assuming you're right, the solution would be to find a function which will tell you whether or not it's suspended.

the problem is I couldn't find any such functions , so the only other logical alternative would be to do a reset on close event. 

though I do agree that finding such function would be the ideal solution.